vertx-web-graphql/src/main/asciidoc/index.adoc

```diff
@@ -199,6 +199,54 @@ Here you can find how to configure the Apollo SubscriptionClient: https://github
 
 IMPORTANT: A subscription `DataFetcher` has to return a `org.reactivestreams.Publisher` instance.
 
+=== GraphQL multipart request
+
+To use https://github.com/jaydenseric/graphql-multipart-request-spec which will allow you to upload one or more files, using a single mutation call.
+
+To enable it, create the {@link io.vertx.ext.web.handler.graphql.GraphQLHandler} with options:
+
+[source,$lang]
+----
+GraphQLHandlerOptions graphQLHandlerOptions = new GraphQLHandlerOptions()
+  .setRequestMultipartEnabled(true);
+----
+
+Next, configure the router with the `BodyHandler`. This handler will parse the multipart HTTP body.
+
+Important: If the router does not have a BodyHandler, the multipart request parser will not be able to handle the `GraphQL` mutation call.
+
+Finally, create the Upload scalar and set it to the `RuntimeWiring`:
+
+[source,$lang]
+----
+GraphQLScalarType uploadScalar = new GraphQLScalarType.Builder()
```

Comment:
Reading this, it seems to me we should have this scalar type in the code and only document how to declare it.